Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -163,7 +175,14 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
         subtitleLabel.setContentCompressionResistancePriority(.streamLow, for: .horizontal)
         timestampLabel.setContentCompressionResistancePriority(.streamRequire, for: .horizontal)
 
-        NSLayoutConstraint.activate(constraintsToActivate)
+        NSLayoutConstraint.activate(
+            constraintsToActivate
```

Comment:
Since `constraintsToActive` now contains only `visualCenterGuide` constraints, maybe we should rename it to `visualCenterGuideConstriant`?

Reply:
makes sense, it will be more consistent anyways